frontend/src/App.js

```diff
@@ -0,0 +1,22 @@
+import { Navigate, BrowserRouter, Routes, Route } from "react-router-dom";
+import Login from "./pages/Login";
+import Signup from "./pages/Signup";
+import { useAuthContext } from "./hooks/useAuthContext";
+
+const App = () => {
+  const {orgUser} = useAuthContext()
+
+  return (
+    <BrowserRouter>
```

Comment:
What's the thinking behind putting the browser router in here instead of in index.js?

Reply:
The  browser router is just a special react component that needs to wrap App so the routing works, it can live in index.js or app.js. I believe on this occasion the MERN stack tutorial had it wrap `<BrowserRouter>` in app.js so it was put there